src/components/Footer.jsx

```diff
@@ -7,12 +7,16 @@ import logo from "../../public/images/navlogo.webp";
 const Footer = () => {
   return (
     <div className="bg-gradient-to-r from-isa-blue-200 to-isa-blue-100 w-full p-2 relative">
-      <div className="flex items-center font-bold text-3xl text-white space-x-3">
-        <Image src={logo} alt="ISA icon" className="h-[45px] w-[45px]" />
+      <div className="flex items-center font-bold text-3xl text-white space-x-3 h-[60px]">
+        <Image src={logo} alt="ISA icon" className="h-[50px] w-[50px]" />
         <div className="whitespace-nowrap">UCR ISA</div>
-        <div className="w-full max-h-full flex space-x-4 place-content-end">
+        <div className="w-full flex place-content-end align-items-center">
```

Comment:
![image](https://github.com/acm-ucr/isa-website/assets/91030482/71b0c97d-fdb3-45c2-aeda-19cbe51f4de8)

looks great on desktop. maybe add more spacing between the social media icons.